components/ChapterCard.tsx

```diff
@@ -34,21 +38,23 @@ const ChapterCard = () => {
         <div className='flex group-hover:invisible w-full justify-between text-grey'>
           <div className='flex flex-col justify-center items-center gap-2'>
             <span className='self-end text-[14px] leading-[17px] font-medium'>
-              Values, Types, and Operators
+              {exercises}
             </span>
           </div>
         </div>
```

Comment:
Do we want to show the exercises here? I thought this card was just a link to the chapter page, where the actual exercise is shown